pkgs/os-specific/linux/util-linux/rtcwake-search-PATH-for-shutdown.patch

```diff
@@ -2,26 +2,68 @@ Search $PATH for the shutdown binary instead of hard-coding /sbin/shutdown,
 which isn't valid on NixOS (and a compatibility link on most other modern
 distros anyway).
 
-  -- nckx <github@tobias.gr>
 --- a/include/pathnames.h
 +++ b/include/pathnames.h
-@@ -53,7 +53,7 @@
+@@ -50,8 +50,8 @@
  #ifndef _PATH_LOGIN
- #define _PATH_LOGIN		"/bin/login"
+ # define _PATH_LOGIN		"/bin/login"
```

Comment:
This looks wrong. Doesn't this turn this into a regular comment instead of a define?

Reply:
Maybe in some future C standard, to woo the Python kids :wink:

I don't understand the reason for inserting a random (harmless) space here though.

Reply:
Yes, I got my python foo mixed up with my C foo here :laughing:

Reply:
@Emantor I patched on util-linux 2.35.1 and exported the diff here. This line is just kept from 2.35.1 and not touched by me.

Reply:
In short, the space after `#` is added in upstream.
See https://github.com/karelzak/util-linux/blob/v2.35.1/include/pathnames.h#L51

Reply:
Weird, but okay.